packages/web-components/src/components/ic-search-bar/test/basic/ic-search-bar.e2e.ts

```diff
@@ -1060,28 +1060,27 @@ describe("ic-search-bar", () => {
     });
 
     expect(focusedElement).toBe("ic-search-bar-input-0-menu-bar");
-  });
-
-  it("menu is still rendered when Shift is pressed while focus on input", async () => {
-    const page = await newE2EPage();
-    await page.setContent(`<ic-search-bar label="Test Label"></ic-search-bar>`);
-    const searchBar = await page.find("ic-search-bar");
-    searchBar.setProperty("options", options);
 
     await page.waitForChanges();
 
-    await focusAndTypeIntoInput("ba", page);
+    await page.keyboard.down("Shift");
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("Tab");
 
     await page.waitForChanges();
 
-    await page.keyboard.press("Shift");
-
-    const menu = await page.find("ic-search-bar >>> ic-menu");
+    focusedElement = await page.evaluate(() => {
+      const el = document
+        .querySelector("ic-search-bar")
+        .shadowRoot.querySelector("ic-text-field");
+      return el.shadowRoot.activeElement.id;
+    });
 
-    expect(await menu.isVisible()).toBeTruthy();
+    expect(focusedElement).toBe("ic-search-bar-input-0");
   });
 
-  it("focuses back onto search bar from menu", async () => {
```

Comment:
Has this test been removed entirely then? and is that okay if so?

Reply:
I've removed this test and added the focusing back onto the search bar from menu into another test - all of the start of the test for reaching the menu was already being done in another test and not intermittently failing so it was half being duplicated here.